frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopgate/terms-and-conditions",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
in other places you changed to version to 1.1.1

Reply:
Sorry I think I didn't end up pushing the updated version for this package file.

Reply:
Sorry, version fixed.